File: springboot-stockapp/src/main/java/com/pascalskillz/model/Stock.java
package com.pascalskillz.model;

import lombok.Data;
import lombok.ToString;

@Data
@ToString
public class Stock {

    private String companyName;
    private String stockSymbol;
    private double currentPrice;
    private double lastPrice;
    private int percentDifference;
    private boolean isNegative;
    private String differenceAmount;

    public Stock(String companyName, String stockSymbol, double lastPrice, double currentPrice){
        this.companyName = companyName;
        this.stockSymbol = stockSymbol;
        this.lastPrice = lastPrice;
        this.currentPrice = currentPrice;
    }

}

File: angular-frontend/src/app/models/stock.ts
export class Stock {
    companyName: string
    stockSymbol: string;
    currentPrice: number;
    lastPrice: number;
    percentDifference: Number;
    isNegative: boolean;
    differenceAmount: any;

    constructor(companyName: string, stockSymbol: string, lastPrice: number, currentPrice: number) {
        this.companyName = companyName;
        this.stockSymbol = stockSymbol;
        this.lastPrice = lastPrice;
        this.currentPrice = currentPrice
    }
}
